content/assignments/techjobs-mvc/_index.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "Assignment 3: TechJobs MVC"
+date: 2023-01-05T13:58:39-06:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Introduction
+
+Your first two tasks as an apprentice went well! You, Blake, and Sally built
+the TechJobs console prototype and then refactored the code to move it to an
+object-oriented format.
+
+After demonstrating the prototype for the Company Team at LaunchCode, it
+received the green light to be fully built out as a web application.
+
+The first step in this process will be to quickly develop a [minimum viable
+product](https://en.wikipedia.org/wiki/Minimum_viable_product), or MVP. The
+goal is to get a functioning web app up and running with as little work as
+possible. That way, additional feedback and testing can be done early in the
+development process. After that, additional behind-the-scenes work will be
+carried out to fully develop the model and data side of the application.
+
+For this next step in the project, you’ll be working with Carly.
+
+{{< rawhtml >}}
+   <img src="pictures/LC-Carly.png" alt="LaunchCode Mentor Carly" width=20% />
+{{< /rawhtml >}}
+
+Carly was once a LaunchCode apprentice as well, so she knows just what
+it’s like to be in your shoes. She’s done some initial work on the
+project and left you some `TODO` tasks that she knows you can handle.
+
+## Learning Objectives
+
+In this project, you’ll show that you can:
+
+1. Read and understand code written by others.
+1. Work within the controller and view portions of a ASP.NET MVC application.
+1. Use Razor syntax to display data within a view.
+1. Create new action methods to process form submission.
```

Comment:
Please match Assignment 0 - how it was broken down into tasks, each with a separate page.  It would help with the continuity of the assignments.

Reply:
Done!

Reply:
Looks Great!